samcli/lib/providers/api_collector.py

```diff
@@ -184,6 +189,18 @@ def get_api(self) -> Api:
         api.stage_name = self.stage_name
         api.stage_variables = self.stage_variables
         api.cors = self.cors
+
+        for authorizers in self._authorizers_per_resources.values():
+            if len(authorizers):
+                message = f"""{os.linesep}AWS SAM CLI does not guarantee 100% fidelity between authorizers locally 
+vs authorizers deployed on AWS. Any application critical behavior should
```

Comment:
I know I made the suggestion, but on re-reading. we could replace "vs" with "and".